.circleci/config.yml

```diff
@@ -123,6 +139,12 @@ jobs:
     steps:
       - build-and-persist-packer-binary:
           GOOS: darwin
+  build_darwin_arm64:
+    executor: golang
+    working_directory: /go/src/github.com/hashicorp/packer
+    steps:
+      - build-and-persist-packer-binary-arm64:
+          GOOS: darwin
```

Comment:
```suggestion
      - build-and-persist-packer-binary:
          GOOS: darwin
          GOARCH: arm64
```

Reply:
( to go with my previous suggestion )

Reply:
@azr this is a good suggestion. The amd64 build commands use `<<go env GOARCH>>` for setting the arch. I think we should also update the command to use `<<parameter.GOARCH>>` and explicitly set `GOARCH=<<parameter.GOARCH>>` in the go build command. Otherwise the parameter is not really being used.